frontend/src/store/user.ts

```diff
@@ -0,0 +1,58 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit'
+
+export interface UserState {
+  userID: string,
+  status: string
+}
+
+
+const url = 'http://localhost:8000/user/logged';
+
+export const setUser = createAsyncThunk(
+  'user/setUser',
+  async () => {
+    return await fetch(url,  {
```

Comment:
suggestion (for the future): use `axios` or some custom service that will set all the headers/config options for communication with the API - it will clean up the requests a bit, since you won't have to define the same thing everytime you're using `fetch()`

https://github.com/axios/axios#creating-an-instance